### PR TITLE
[DE][Exceptions] Allow duplicate case sensitive values for match_any

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3992.

Previews:
- [Add exceptions to a rule](https://security-docs_4023.docs-preview.app.elstc.co/guide/en/security/master/add-exceptions.html#detection-rule-exceptions): Added note to step 3c.
- [Add Elastic Endpoint exceptions](https://security-docs_4023.docs-preview.app.elstc.co/guide/en/security/master/add-exceptions.html#endpoint-rule-exceptions): Modified step 2 and added an optional step 3 (adding a comment to the exception).

### CODE REVIEW

File: docs/detections/add-exceptions.asciidoc [hunk 1]
```diff
@@ -178,10 +178,13 @@ image::images/endpoint-add-exp.png[]
 
 . If required, modify the conditions.
 +
-NOTE: Refer to <<ex-nested-conditions>> for more information on when nested conditions are required.
-+
-NOTE: Fields with conflicts are marked with a warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]). Using these fields might cause unexpected exceptions behavior. For more information, refer to <<rule-exceptions-field-conflicts, Troubleshooting type conflicts and unmapped fields>>.
+[NOTE] 
+======
+* Fields with conflicts are marked with a warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]). Using these fields might cause unexpected exceptions behavior. For more information, refer to <<rule-exceptions-field-conflicts, Troubleshooting type conflicts and unmapped fields>>.
+* Identical, case-sensitive values are supported for the `is one of` and `is not one of` operators. For example, if you want to match the value `Windows` and `windows`, add both values to the **Value** field.
```
Comment (natasha-moore-elastic) [inline docs/detections/add-exceptions.asciidoc:184]:
```suggestion
* The `is one of` and `is not one of` operators support identical, case-sensitive values. For example, if you want to match the values `Windows` and `windows`, add both values to the **Value** field. 
```

File: docs/detections/add-exceptions.asciidoc [hunk 2]
```diff
@@ -89,13 +89,13 @@ NOTE: Some characters must be escaped with a backslash, such as `\\` for a liter
 +
 IMPORTANT: Using wildcards can impact performance. To create a more efficient exception using wildcards, use multiple conditions and make them as specific as possible. For example, adding conditions using `process.name` or `file.name` can help limit the scope of wildcard matching.
 
-  .. *Value*: Enter the value associated with the *Field*. To enter multiple values (when using `is one of` or `is not one of`), enter each value, then press **Return**.
-
-+
+.. *Value*: Enter the value associated with the *Field*. To enter multiple values (when using `is one of` or `is not one of`), enter each value, then press **Return**. 
 +
-In the example below, the exception was created from the Rules page and prevents the rule from generating alerts when the `svchost.exe` process runs on hostname `siem-kibana`.
+NOTE: Identical, case-sensitive values are supported for the `is one of` and `is not one of` operators. For example, if you want to match the value `Windows` and `windows` add both values to the **Value** field.
```
Comment (natasha-moore-elastic) [inline docs/detections/add-exceptions.asciidoc:94]:
```suggestion
NOTE: The `is one of` and `is not one of` operators support identical, case-sensitive values. For example, if you want to match the values `Windows` and `windows`, add both values to the **Value** field. 
```

**Review by natasha-moore-elastic — APPROVED** [review]
Just one minor suggestion, otherwise LGTM!

**Review by yctercero — APPROVED** [review]
LGTM- thank you!

**Pull request merged**